packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -89,6 +103,18 @@ class Checkbox extends StatefulWidget {
   /// Defaults to accent color of the current [Theme].
   final Color activeColor;
 
+  /// If true the checkbox's [value] can be true, false, or null.
+  ///
+  /// Checkbox displays a dash when its value is null.
+  ///
+  /// When a tri-state checkbox is tapped its [onChanged] callback will be
+  /// applied to true if the current value is null or false, false otherwise.
```

Comment:
not sure what "applied to" means here either

Reply:
https://en.wikipedia.org/wiki/Apply